src/lib/button/button.ts

```diff
@@ -52,8 +55,10 @@ const BUTTON_HOST_ATTRIBUTES = [
 export class MatButtonBase {
   constructor(public _elementRef: ElementRef) {}
 }
-export const _MatButtonMixinBase = mixinColor(mixinDisabled(mixinDisableRipple(MatButtonBase)));
 
+export const _MatButtonMixinBase:
```

Comment:
These mixins are getting more and more inconvenient to consume :/

Reply:
I'm going to file an issue with TypeScript to see if they can be smarter about keeping the original import paths